scripts/managed-services/get-kafka-instance-state/get-kafka-instance-state.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+function main() {
+  local kafkaId=$1
+  local managedKafkaCR=$(oc -n kafka-$kafkaId get managedkafka -o name)
+  local kafkaCR=$(oc -n kafka-$kafkaId get kafka -o name)
+  local kafkaStatefulSet=$(oc -n kafka-$kafkaId get statefulset -l app.kubernetes.io/name=kafka -o name)
+  local timestamp=$(date +%s)
+  local inspectDir="inspect.kafka-$kafkaId.$timestamp"
+  local namespaceCheck=$(oc get namespace kafka-$kafkaId)
+
+  oc -n kafka-$kafkaId adm inspect ns/kafka-$kafkaId $managedKafkaCR $kafkaCR --dest-dir $inspectDir > /dev/null 2>&1
+
+  oc exec -n kafka-$kafkaId $kafkaStatefulSet -c kafka -- sh /opt/kafka/bin/kafka-topics.sh --bootstrap-server localhost:9096 --list > ${inspectDir}/kafka-topics.txt 2>&1
+
+  oc exec -n kafka-$kafkaId $kafkaStatefulSet -c kafka -- sh /opt/kafka/bin/kafka-consumer-groups.sh --bootstrap-server localhost:9096 --all-groups --describe  > ${inspectDir}/kafka-consumer-groups.txt 2>&1
+
+  oc exec -n kafka-$kafkaId $kafkaStatefulSet -c kafka -- sh /opt/kafka/bin/kafka-acls.sh --bootstrap-server localhost:9096 --list > ${inspectDir}/kafka-acls.txt 2>&1
+
+  tar zcf - ./${inspectDir}
+}
+
+main $1
```

Comment:
This won't hold the kafkaid, the way the arguments work is by adding an envvar with the corresponding name to the pod